### PR TITLE
Update Set-SPOSite.md

### PR DESCRIPTION
**Pull request by panjkov**
Improvements in formatting and readability, clarification and fixed link in description of -SharingDomainRestrictionMode

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 1]
```diff
@@ -667,17 +667,14 @@ Accept wildcard characters: False
...
 ### -SharingDomainRestrictionMode
 
-Specifies the external sharing mode for domains.
+Specifies the sharing mode for external domains.
 
-The following values are:
+Possible values are:
+- None - Do not restrict sharing by domain
+- AllowList - Sharing is allowed only with external users that have account on domains specified within -SharingAllowedDomainList
+- BlockList - Sharing is allowed with external users in all domains expect in domains specified within -SharingBlockedDomainList
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:675]: Did you mean "except" here?

**Review by get-itips — COMMENTED** [review]
Just a minor detail. Thanks

**Review by get-itips — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md (hunk 1 above)
Comment (panjkov) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md]: Of course it should be "except" :) I just changed it in the branch, PR should be updated.

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks.

**Review by tseward — APPROVED** [review]

**Pull request merged**